docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+# Default Reference Types
+
+We've mentioned throughout this tutorial what the default reference capabilities are for types and classes. However, there's a good chance those defaults passed over your head while you were focused on understanding capabilities themselves.
+
+First, there is nothing saying you have to use default capabilities. Especially as you start writing your first Pony programs, you'll probably find it very educational to explicitly specify the capabilities for all your classes, actors, functions, parameters and fields. However, it's not _writing_ code that makes default capabilities confusing; it's _reading_ that code. Hopefully keeping this summary open will help you navigate pony source code until you've become used to the defaults!
+
+## Reference type summary
+
+We go into detail of the default types below, but here's a table summarizing them:
+
+| Item                    | Default Capability | Explanation                                                             |
+| ----------------------- | ------------------ | ----------------------------------------------------------------------- |
+| primitive               | `val`              | Primitives are immutable and can never be anything but `val`            |
```

Comment:
While primitives are immutable, reference capabilities apply to references. One could in theory have a `P tag` reference to some primitive `P`.

Perhaps: are always constructed as `val`

Reply:
Changed it as this and added a note about `tag` to the text below.